.github/workflows/_tests.yml

```diff
@@ -41,9 +41,13 @@ jobs:
         target: ${{fromJson( inputs.changed )}}
     steps:
       - uses: actions/checkout@v3
+      - name: Setup integration environment
+        run: scripts/setup-integration.sh
```

Comment:
Let's rename this because it is **not** integration tests

Reply:
`scripts/setup-stack.sh`

Reply:
We should also rename `./bin/tests-local.sh`

Reply:
`/Users/julien/repos/unlock/tests/bin/tests-local.sh` and `/Users/julien/repos/unlock/tests/bin/tests.sh` should be split between the "setup" part (that moves to `/unlock/scripts`) and the "runner" part that stays here.